specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -0,0 +1,19755 @@
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "List continuous web jobs for an app, or a deployment slot.",
+        "description": "List continuous web jobs for an app, or a deployment slot.",
+        "operationId": "WebApps_ListContinuousWebJobs",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Site name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContinuousWebJobCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/continuouswebjobs/{webJobName}": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets a continuous web job by its ID for an app, or a deployment slot.",
+        "description": "Gets a continuous web job by its ID for an app, or a deployment slot.",
+        "operationId": "WebApps_GetContinuousWebJob",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Site name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "webJobName",
+            "in": "path",
+            "description": "Name of Web Job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Found continuous web job.",
+            "schema": {
+              "$ref": "#/definitions/ContinuousWebJob"
+            }
+          },
+          "404": {
+            "description": "Continuous web job does not exist."
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Delete a continu
```

Comment:
[informational] new enum value added (`BackupRestoreOperationType`)

@ruslany please consider changing this to `true` (that applies in general) since that way, changes to the set of enum values are not SDK breaking in the future.